src/runtime/components/InputMenu.vue

```diff
@@ -172,6 +171,8 @@ onMounted(() => {
     :display-value="displayValue"
     :filter-function="filterFunction"
     :class="ui.root({ class: props.class })"
+    @update:model-value="emitFormChange()"
+    @keydown.enter="$event.preventDefault()"
```

Comment:
@romhml Can you explain what this is for?

Reply:
Pressing enter on an Input inside a form triggers the form submission / validation. This clashes with the InputMenu where pressing enter should only select the first option.